app/main.py

```diff
@@ -16,4 +16,16 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+#  write your code here
+sorted_variables = {"mutable": [], "immutable": []}
+
+mutable_vars = [my_favourite_films, marks, collection_of_coins]
+immutable_vars = [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+
+for element in mutable_vars:
```

Comment:
these loops don't make sense. you can do it easily. just put data in dict immediately

Reply:
delete useless code